spec/system/account_system_spec.rb

```diff
@@ -6,12 +6,30 @@
     visit subject
   end
 
-  it "should change a user name" do
+  it "should change an user name" do
```

Comment:
This is a weird English anomaly, but because the "u" here is a "hard u" (when pronounced, it sounds like "you"), it leads with the sound of a consonant, so you would use the regular "a" article. When written, it looks like it should be "an", though, so I completely understand where you're coming from! :)

Not a blocker and I'll approve this anyways -- just thought you should know. :)

Reply:
Huh, TIL! thanks for the info! 😄